deploy/crds/ocmagent.managed.openshift.io_ocmagents_crd.yaml

```diff
@@ -29,10 +29,59 @@ spec:
           metadata:
             type: object
           spec:
-            description: OCMAgentSpec defines the desired state of OCMAgent
+            description: OcmAgentSpec defines the desired state of OcmAgent
+            properties:
+              environment:
```

Comment:
I've left a more detailed comment in the CR design document around this (and sorry that I didn't catch it sooner) but I think the value of this should instead be the actual OCM base URL (ie. "https://api.openshift.com") and let app-interface populate what it thinks the correct value should be.. so that we don't put in logic into OAO around determining an OCM URL that could break anytime someone changes the OCM URL in app-interface.